### ISSUE TITLE
Dropout Seed Value in Siamese Networks

### ISSUE THREAD

**Issue by lschaupp** [keras-team/keras#10392]
Hey,

Following question: 

I'm implementing a siamese network with dropout functionality and I'm wondering
about the non-deternism of the layer between shared weights. (as two data streams would 
yield two different results which should not be the case)
I've figured that I would set the seed value of dropout value to a fixed value to use the same values
between two data streams.
One question comes up though: Using the same seed though, would result that always the same dropout  will be applied to the network (over the time steps)? If this is the case, I would conclude that an update of the seed parameter is needed during each training step?! Is this a correct guess?

Thanks ;)
